enhancements/windows-containers/container-runtime-containerd.md

```diff
@@ -0,0 +1,246 @@
+
+* If the cluster is upgraded, and the new version introduces an issue due to containerd
+  that should be addressed because Docker runtime support won't be available in kubelet. We work closely with the
+  containerd open source community and try to resolve it as soon as possible. We aim to fix any critical issue in the
+  upstream first and then update the containerd submodule to point to the latest containerd upstream version. If there
+  is any critical issue that needs to be addressed as soon as possible, as an exception we carry patch(es) and then
+  follow up with upstream.
+* As we don't support downgrade, reverting to an older version is not possible. To overcome
+  any issues, we introduce this feature as part of the 5.0 community operator so that this feature
+  can be well tested before we make it default in WMCO 6.0 release. We will use two test jobs to test all e2e test cases
+  in the 4.10 community branch(OKD) and 4.10 branch(ccm OCP)to capture any regression.
+  This will help us to identify any issues well before containerd becomes default runtime.
+* containerd doesn't support image-pull-progress-deadline as of now. There is a PR
+  https://github.com/containerd/containerd/pull/6150 work in progress. Until this
+  gets merged, if Windows image pull takes more time than the default value, we might
+  run into to image pull timeout error. To overcome this issue, Customers can bake required base container images into
+  Windows VM image. If customers still notice timeout error, they should log in to Windows VM and install `ctr or crictl`
+  command-line tool. Users can use one of these tools to pull the container image manually. Upon successful image download,
+  the user can create pods.
+* Currently Windows_exporter has been used to collect metrics from Windows node. We do
+  have containerd support in [Windows exporter v0.16.0](https://github.com/prometheus-community/Windows_exporter/releases/tag/v0.16.0)
+  As of now, there is no feature parity mismatch between Docker and containerd.
+
+#### Removing a deprecated feature
+Once containerd becomes the default runtime, Docker will no longer be needed in Kubernetes stack. This is discussed in
+design section.
+
+### API Extensions
+
+N/A
+
+### Operational Aspects of API Extensions
+
+N/A
+
+#### Failure Modes
+
+Failure to start containerd service in Windows node will put the node into 'not ready' state. We will not be able to
+schedule any pods into the Windows node.
+
+#### Support Procedures
+
+If containerd service doesn't get started, kubelet will not be in ready state. Windows node will not be able to schedule
+any pods. Containerd logs should be collected via must-gather script. We will check containerd service status and update the
+WMCO event log if the service is not up and running. In addition, we will be generating a Kubernetes event.
+
+### Test Plan
+
+We will use one OKD job and one ccm OCP job to test the WMCO community operator. We will pass the WMCO CLI option
```

Comment:
We are not testing the "community" operator but testing WMCO against OKD and OCP. So please remove `community` from this sentence and say `...test WMCO with the  containerd as the runtime using the --runtime CLI option`. You can then remove `We will pass the WMCO CLI option...` sentence. You should also say this is for release-4.10 and in release-4.11 we will migrate all jobs to containerd.